{
  "comments": [
    {
      "key": {
        "uuid": "268c5842_d499cb9f",
        "filename": "/COMMIT_MSG",
        "patchSetId": 1
      },
      "lineNbr": 9,
      "author": {
        "id": 1080813
      },
      "writtenOn": "2017-03-09T08:30:03Z",
      "side": 1,
      "message": "I\u0027m not sure about the reason.\n\nShould the errno usually reflect the status of the latest (last) action?",
      "range": {
        "startLine": 9,
        "startChar": 54,
        "endLine": 9,
        "endChar": 71
      },
      "revId": "ea2b0d19553048d9003142301d4c874375934195",
      "serverId": "85c56323-6fa9-3386-8a01-6480fb634889",
      "unresolved": false
    }
  ]
}